pkg/controller/elasticsearch/certificates/transport/trust.go

```diff
@@ -0,0 +1,36 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package transport
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	v1 "github.com/elastic/cloud-on-k8s/v2/pkg/apis/elasticsearch/v1"
+	"github.com/elastic/cloud-on-k8s/v2/pkg/controller/common/certificates"
+	"github.com/elastic/cloud-on-k8s/v2/pkg/utils/k8s"
+)
+
+func MaybeRetrieveAdditionalCAs(ctx context.Context, client k8s.Client, elasticsearch v1.Elasticsearch) ([]byte, error) {
```

Comment:
Do we need to watch the `configMap`?

Reply:
Oh yes! I changed the implementation. Originally I had the config map mounted directly into the Pod and thought Elasticsearch would take care of watching the contents changing. But now that I am concatenating it is actually necessary to watch it.